rotkehlchen/serialization/deserialize.py

```diff
@@ -542,15 +554,15 @@ def deserialize_ethereum_transaction(
         else:
             timestamp = deserialize_timestamp(data['timeStamp'])
 
-        from_address = deserialize_ethereum_address(data['from'])
         is_empty_to_address = data['to'] != '' and data['to'] is not None
         to_address = deserialize_ethereum_address(data['to']) if is_empty_to_address else None
         value = read_integer(data, 'value', source)
 
         if internal:
+            trace_id = trace_id if trace_id is not None else int(data.get('traceId', '0'))
```

Comment:
Btw, I think that you don't need a default argument for `data.get('traceId', '0')`. Previously it was used without it and everything was working fine